runtimes/cloudformation/go.mod

```diff
@@ -3,42 +3,42 @@ module github.com/falcosecurity/kilt/runtimes/cloudformation
 go 1.19
 
 require (
-	github.com/Jeffail/gabs/v2 v2.6.1
-	github.com/aws/aws-lambda-go v1.37.0
-	github.com/aws/aws-sdk-go v1.44.178
-	github.com/falcosecurity/kilt/pkg v0.0.0-20230111165949-b36cdd622de1
-	github.com/google/go-containerregistry v0.12.1
-	github.com/rs/zerolog v1.28.0
+	github.com/Jeffail/gabs/v2 v2.7.0
+	github.com/aws/aws-lambda-go v1.41.0
+	github.com/aws/aws-sdk-go v1.44.284
+	github.com/falcosecurity/kilt/pkg v0.0.0-20230606123839-2e4c434d5d88
+	github.com/google/go-containerregistry v0.15.2
+	github.com/rs/zerolog v1.29.1
 	github.com/stretchr/testify v1.8.1
 	github.com/yudai/gojsondiff v1.0.0
 )
 
 require (
-	github.com/containerd/stargz-snapshotter/estargz v0.12.1 // indirect
+	github.com/containerd/stargz-snapshotter/estargz v0.14.3 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/docker/cli v20.10.20+incompatible // indirect
-	github.com/docker/distribution v2.8.1+incompatible // indirect
-	github.com/docker/docker v20.10.24+incompatible // indirect
+	github.com/docker/cli v24.0.2+incompatible // indirect
+	github.com/docker/distribution v2.8.2+incompatible // indirect
```

Comment:
I suppose this is the important line? ;)

Reply:
I sure hope so!